addons/l10n_zm/__manifest__.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+{
+    "name": "Zambia - Accounting",
+    "countries": ["zm"],
+    "version": "1.0.0",
+    "category": "Accounting/Localizations/Account Charts",
+    "license": "LGPL-3",
+    "description": """
+This is the basic Zambian localization necessary to run Odoo in ZM:
+================================================================================
+    - Chart of accounts
+    - Taxes
+    - Fiscal positions
+    - Default settings
+    - Financial (balance sheet & profit and loss) and Tax reports
```

Comment:
BS and PL are only in Enterprise version so let's not bait users and put it only in the Enterprise manifest.